pxr/usd/plugin/usdMtlx/utils.cpp

```diff
@@ -297,7 +297,7 @@ _ReadFromAsset(mx::DocumentPtr doc, const ArResolvedPath& resolvedPath,
                            newReadOptions);
         };
 
-    mx::readFromXmlString(doc, s, &readOptions);
+    mx::readFromXmlString(doc, s, searchPath, &readOptions);
```

Comment:
Is this purely to update the API, or will there be any behaviour change here.
If syntatical then should this instead be an empty search path ? Seems passing
`searchPath` is correct though. Just wondering.

Reply:
Just updating to [the latest API](https://github.com/materialx/MaterialX/pull/707). No behavior change.